packages/react-svg-core/src/index.ts

```diff
@@ -1,15 +1,13 @@
-import Svgo from "svgo";
 import { transformSync as babelTransform } from "@babel/core";
 import plugin from "babel-plugin-react-svg";
-
+import { optimize } from "svgo";
 import { validateAndFix } from "./svgo";
 
+
 // SVGO Optimize
-export function optimize(opts: any = {}): (content: string) => Promise<string> {
+export function optimize(opts: any = {}) {
   opts = validateAndFix(opts);
```

Comment:
Config format is different now. So validateAndFix might need to be changed as well.